skgrid/run_train.py

```diff
@@ -31,41 +31,12 @@
 from sklearn.gaussian_process.kernels import RationalQuadratic
 from sklearn.gaussian_process.kernels import WhiteKernel
 
-
-classifers = {
```

Comment:
What is the reasoning for moving this code to a config file? These link to the actual classes, which can't be serialized to JSON